nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/SleepWork.java

```diff
@@ -40,6 +40,8 @@
 
     protected boolean idempotent = true;
 
+    protected boolean coalescing = false;
```

Comment:
This is false by default, no need to set it.

Reply:
This isn't actually resolved, although no big deal.